data/base-spells.yaml

```diff
@@ -865,6 +865,17 @@ spell_data:
     abbrev: IOTS
     mana: 300
     ritual: true
+    recast: 0
```

Comment:
If we cast IOTS on a different stat than the currently active IOTS.. instant death. Classic moon mage